runtime/parser/keyword.go

```diff
@@ -129,6 +141,7 @@ var softKeywords = []string{
 	KeywordSet,
 	KeywordAll,
 	KeywordView,
+	keywordTo,
```

Comment:
```suggestion
	keywordRemove,
```

Reply:
Hmm, I'm not sure it can be actually, because the soft keyword test then fails. If we see a `remove = 3` assignment, the parser starts parsing this as a `remove` statement rather than as an assignment. Unless I am misunderstanding then we need to make `remove` a hard keyword, and thus technically attachments would be a breaking change?

Reply:
Ah, I see. I'll refamiliarize myself with how we handle hard vs soft keywords at the moment and will see how we can handle these cases. We might have to improve the parsing function for the remove statement a bit

Reply:
Going to handle this in a different PR, as it's not related to entitlements.

Reply:
We will address the attachment related code changes in a separate PR